app/models.py

```diff
@@ -53,7 +53,6 @@ class Bit(models.Model):
 
     created_on = models.DateTimeField(auto_now_add=True)
     last_modified = models.DateTimeField(auto_now=True)
-    posted_by = models.ForeignKey(Member, related_name='bit', on_delete=models.CASCADE)
```

Comment:
Why don't we wanna know who added the bit?
Am I missing something here?

Reply:
I believe this wasn't part of the original code pushed yesterday and you pushed this today morning.
Also, bit is linked to curriculum which has the author.